src/Plugin.php

```diff
@@ -50,11 +50,15 @@ public static function wp_enqueue_scripts() {
   }
 
   /**
+   * Add image disclaimer text as title of wrapping link.
+   * Text displayed in front-end via CSS pseudo element.
+   *
    * @implements woocommerce_single_product_image_thumbnail_html
    */
   public static function woocommerce_single_product_image_thumbnail_html($content = NULL) {
-    $disclaimer = '<span class="disclaimer-overlay">' . __('Photo may show optional equipment not included in delivery.', Plugin::L10N) . '</span>';
-    return preg_replace('/(<img[^>]+>(?:<\/img>)?)/i', '$1' . $disclaimer, $content);
+    $match = 'class="woocommerce-product-gallery__image"';
+    $disclaimer = ' title="' . __('Photo may show optional equipment not included in delivery.', Plugin::L10N) . '"';
+    return str_replace($match, $match . $disclaimer, $content);
```

Comment:
What happens if the image thumbnail already have a title? 🤔

Reply:
`.woocommerce-product-gallery__image` is not actually the `<img>` tag but the `<div>` wrapping the image and link. So it won't ever get a `title` added to it.
Go WooCommerce 👏